back/app/serializers/web_api/v1/cosponsors_initiative_serializer.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class WebApi::V1::CosponsorsInitiativeSerializer < WebApi::V1::BaseSerializer
+  attributes :status, :user_id, :initiative_id
+end
```

Comment:
Not really needed yet, as we're not using this serializer in this PR.

Reply:
Will it ever be used? If not anticipated, then we don't need to add it.